### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by slnw** [dedoc/scramble#280] [General]
@romalytvynenko -
It used to get multiple updates over time and prompt answers. What is going on? Maybe you should open the website SaaS offering/sponsers and get a few paid developers to contribute code?

**Comment by romalytvynenko** [comment]
@slnw I'm still going to work on the project, but I want to give it full focus. Which I cannot do at this moment. 

What you say about saas or sponsors is nice to hear, for sure :) 

For quicker answers you can try asking the community on Discord, there is more activity.
